copyfiles

```diff
@@ -26,9 +26,12 @@ args.alias('a', 'all')
   .alias('s','soft')
   .boolean('s')
   .describe('s', 'do not overwrite destination files if they exist')
-  .usage('$0 [options] inFile [more files ...] outDirectory').help()
+  .boolean('follow')
```

Comment:
can we get a short alias for this?

Reply:
sure, but `f` was already taken by `flatten`. Any suggestion?

Reply:
upper case F ?

Reply:
alias added :D